### PR TITLE
Clan command added

### PR DESCRIPTION
**Pull request by PhillypHenning**
New Clan command has beed added to spirit that will display various stats about a clan 

## Context
https://trello.com/c/eXZM13qf

## How Has This Been Tested?
I have tested it locally but it requires further testing.

## Screenshots (if appropriate):
image

## Types of changes
Added new Clan command to Spirit that outputs and stylizes various available clan stats such as;
Name | About | Level | EXP | Milestones

## Checklist:
- [X] My code follows the code style of this project.
- [X] My change requires a change to the documentation.
- [] I have updated the documentation accordingly. (I will be submitting an updated)

### CODE REVIEW

**Comment by jgayfer** [comment]
@PhillypHenning Sorry for the delay, but I feel I've lost context on where this is at. I was under the impression that you mentioned you weren't finished, but I haven't seen anything in awhile so I'm wondering if I misunderstood.

TL;DR - What's the status on this? :smiley_cat:

File: cogs/clan.py [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+
+        if res['ErrorCode'] != 1:
+            await manager.send_message("Sorry I can't seem to retrieve that clan right now")
+            return await manager.clean_messages()
+
+        group_id = res['Response']['results'][0]['member']['groupId']
+
+        try:
+            ms = await self.bot.destiny.api.get_weekly_milestones(group_id)
+            ms_def = await self.bot.destiny.api.get_weekly_milestone_definitions(ms['Response']['milestoneHash'])
```
Comment (jgayfer) [inline cogs/clan.py:72]: `get_weekly_milestone_definitions` is not a part of Pydest, so this line fails. Perhaps you meant to use `get_milestone_definitions` instead?
